package.json

```diff
@@ -45,6 +45,7 @@
     "sinon": "4.0.1"
   },
   "peerDependencies": {
+    "aws-sdk": "2.131.0",
```

Comment:
This should not be pinned (!)

Reply:
I agree, but it was pinned before so I kept it the same.

Reply:
Ahhh ok great point. I'll change it.